app/models/product.rb

```diff
@@ -15,6 +15,9 @@ class Product < ActiveRecord::Base
   has_many   :price_policies
   has_many   :training_requests, dependent: :destroy
 
+  cattr_accessor(:types) { ConfigurableArray.new([Instrument, Item, Service, Bundle]) }
+  cattr_accessor(:mergeable_types) { ConfigurableArray.new([Instrument, Item, Service, Bundle]) }
```

Comment:
Hmm, there was something weird with this if I disabled the secure_rooms engine. I was getting a circular reference what autoloading Instrument. Switching back to the class methods fixed it.